tools/automation/cache/buildNewCache.sh

```diff
@@ -0,0 +1,12 @@
+bundle install
```

Comment:
Is it worth adding the normal shebang/ex/pipefail incantations here? 👀 

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
I will, however it has no real value as this should not really be run outside the docker call.